### ISSUE TITLE
"Expected user" feature of bash script is annoying

### ISSUE THREAD

**Issue by fiadliel** [gilt/nova#21]
On my machine (with Docker for Mac), I can run docker fine as my own UID.

I disagree that there should be an "expected user" feature that thinks I should run this command as root. If there has to be such a feature, it should only report a problem if docker can't connect to the shared socket (/var/run/docker.sock).

Editing the script is not a reasonable workaround in general.

Before I open a PR, adding this issue for discussion (since it was considered useful enough to add by the writer of the feature).

**Comment by fabiocognigni** [comment]
+1 on removing [that check on expecting root as the user](https://github.com/gilt/nova/blob/65250e7433e4a35117f7bd7cbeae04a840319ff9/bin/nova#L10) . cc/ @rohitst

**Comment by rohitst** [comment]
Yeah, it's a source of confusion at least. I think adding documentation (on this) and on running this the 'docker' way would be great as well.
